script/build.js

```diff
@@ -32,8 +32,8 @@ const minimumNpmVersion = '3.8.9';
 const minimumNodeVersion = '4.4.7';
 // Make sure git pre-commit hooks are installed
 ['pre-commit'].forEach(hook => {
-  const src = `../hooks/${hook}`;
-  const dest = `../.git/hooks/${hook}`;
+  const src = path.join(__dirname, `../hooks/${hook}`);
```

Comment:
@aub Apparently the linking code was broken at some point. I removed my git hook and it wasn't replacing it, because it was looking for the file in a place that didn't exist. This fixes it.